speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -661,6 +649,19 @@ void* ResearcherMenuInit(void* p_nextMenu)
     // Set default userId
     GfuiEditboxSetString(s_scrHandle, m_userIdControl, m_userId);
 
+    // Retrieves the saved user xml file, if it doesn't exist the settings are already initialized in ResearcherMenuInit
+    char buf[512];
+    sprintf(buf, "%s%s", GfLocalDir(), RESEARCH_FILEPATH);
+    if (GfFileExists(buf))
+    {
+        void* param = GfParmReadFile(buf, GFPARM_RMODE_STD);
+        // Initialize settings with the retrieved xml file
+        LoadConfigSettings(param);
```

Comment:
The force feedback option can also be changed in the force feedback menu. Changing it there doesn't update it in the researchermenu.